src/ui/flutter_app/lib/screens/game_page/board.dart

```diff
@@ -51,10 +51,9 @@ class Board extends StatelessWidget {
           child: Text(
             squareDesc[index],
             style: const TextStyle(
+              color:
```

Comment:
If it has demanding performance requirements, it can be optimized to display the transparent grid (coordinate notations) only when the screen reader's support switch is turned on.

Reply:
The thing is that the grid get's build even though the thing is disabled (it will be build but not displayed. 

I rather want to only build it when used...